core-developers/experts.rst

```diff
@@ -77,6 +77,8 @@ collections           rhettinger*
 collections.abc       rhettinger*, stutzbach^
 colorsys
 compileall            carljm
+compression
```

Comment:
I don't think we need `compression` on its own?

Let's move the others (`lzma`, `bz2`, `gzip` and `zlib`) under `compression.`.

https://docs.python.org/3.14/whatsnew/3.14.html#pep-784-adding-zstandard-to-the-standard-library